kdumpctl

```diff
@@ -1668,6 +1672,137 @@ _should_reset_crashkernel() {
 	[[ $(kdump_get_conf_val auto_reset_crashkernel) != no ]] && systemctl is-enabled kdump &> /dev/null
 }
 
+clear_kdump_test_status()
+{
+	echo > $KDUMP_STATUS
+}
+
+# Get the latest dump's info from the dump target in the format:
+# /var/crash 127.0.0.1-2024-05-01-15:54:29/vmcore 1714550071
+#
+# return: 0 on success, 1 on failure
+get_latest_dump()
+{
+	local _latest_record _target
+	local _path=$(get_save_path)
+	if is_raw_dump_target; then
+		dinfo "Unable to check for raw dump. Manual verification of vmcore collection required."
+		return 1
+	elif is_ssh_dump_target; then
+		_latest_record=$(ssh -i "${OPT[sshkey]}" -o BatchMode=yes "${OPT[_target]}" \
+				"cd $_path && \
+				find . -name "vmcore" -or -name "vmcore.flat" \
+				| xargs stat -c %n\ %Y 2> /dev/null \
+				| sort -k2 | sed 's|^\./||' \
+				| tail -1")
```

Comment:
ditto here and other same code, maybe create a function for these search and call them for different cases, ssh, nfs.  And it would be good to add a brief code comment about the purpose of the function.   Otherwise if an incomplete vmcore detected,  it can be regarded as tested, but  it would be good to report it in the warning/notification.